internal/pkg/ecslogging/task.go

```diff
@@ -109,13 +116,13 @@ func (t *TaskClient) allTasksStopped() (bool, error) {
 	return stopped, nil
 }
 
-// The `StartedAt` field for the tasks shouldn't be nil.
-func earliestStartTime(tasks []*task.Task) time.Time {
-	earliest := *tasks[0].StartedAt
+func (t *TaskClient) logStreamNamesFromTasks(tasks []*task.Task) (logStreamNames []string, err error) {
 	for _, task := range tasks {
-		if task.StartedAt.Before(earliest) {
-			earliest = *task.StartedAt
+		id, err := ecs.TaskID(task.TaskARN)
+		if err != nil {
+			return nil, fmt.Errorf("parse task ID from ARN %s", task.TaskARN)
 		}
+		logStreamNames = append(logStreamNames, fmt.Sprintf(fmtLogStreamName, t.GroupName, id))
 	}
-	return earliest
+	return
```

Comment:
lol you love these named return variables with bare returns, I think it's perfectly readable here but for longer functions consider this recommendation:
> Naked returns are okay if the function is a handful of lines. Once it's a medium sized function, be explicit with your return values. 

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters